db_upgrade_scripts/mosip_regprc/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -18,11 +18,60 @@ ALTER TABLE regprc.reg_bio_ref DROP CONSTRAINT IF EXISTS fk_regref_reg CASCADE;
 ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT IF EXISTS fk_rlostd_reg CASCADE;
 ALTER TABLE regprc.registration_transaction DROP CONSTRAINT IF EXISTS fk_regtrn_reg CASCADE;
 
-\ir ../ddl/regprc-additional_info_request.sql
-
-\ir ../ddl/regprc-anonymous_profile.sql
-
-\ir ../ddl/regprc-reg_verification.sql
+CREATE TABLE regprc.additional_info_request(
+	additional_info_process character varying(64),
+	reg_id character varying(39),
+	workflow_instance_id character varying(36),
+	timestamp timestamp,
+	additional_info_iteration integer,
+	additional_info_req_id character varying(256),
+	CONSTRAINT pk_addl_info_req PRIMARY KEY (workflow_instance_id , additional_info_req_id)
+
+);
+
+GRANT SELECT, INSERT, TRUNCATE, REFERENCES, UPDATE, DELETE
+   ON regprc.additional_info_request
+   TO regprcuser;
+
+CREATE TABLE regprc.anonymous_profile(
```

Comment:
same as before for all create ? @vishwa-vyom what do you want?

Reply:
this is the new table added as part of 1.2.0.1 and not present in 1.1.5.5.